targets/python/make.js

```diff
@@ -0,0 +1,407 @@
+
+        //if (property.collection === "array")
+        //    return "List<" + basicType + "> " + property.name;
+        //else if (property.collection === "map")
+        //    return "Dictionary<string," + basicType + "> " + property.name;
+        //else
+        //    throw "Unknown collection type: " + property.collection + " for " + property.name + " in " + datatype.name;
+    }
+    else {
+        //basicType = getPropertyCsType(property, datatype, true);
+        //return basicType + " " + property.name;
+    }
+    return property.name;
+}
+
+function getAuthParams(apiCall) {
+    if (apiCall.url === "/Authentication/GetEntityToken")
+        return "authKey, authValue";
+    if (apiCall.auth === "EntityToken")
+        return "\"X-EntityToken\", EntityToken";
+    if (apiCall.auth === "SecretKey")
+        return "\"X-SecretKey\", DeveloperSecretKey";
+    else if (apiCall.auth === "SessionTicket")
+        return "\"X-Authorization\", PlayFabSettings.ClientSessionTicket";
+    return "None, None";
+}
+
+function getRequestActions(tabbing, apiCall) {
+    if (apiCall.result === "LoginResult" || apiCall.request === "RegisterPlayFabUserRequest")
+        return tabbing + "request[\"TitleId\"] = PlayFabSettings.TitleId or request.TitleId\n"
+            + tabbing + "if request[\"TitleId\"] == None:\n"+tabbing+"    raise Exception(PlayFabExceptionCode.TitleNotSet, \"Must be have TitleId set to call this method\")\n";
+    if (apiCall.auth === "EntityToken")
+        return tabbing + "if PlayFabSettings.EntityToken == None:\n "+tabbing+"    raise Exception(PlayFabExceptionCode.EntityTokenNotSet, \"Must call GetEntityToken before calling this method\")\n";
+    if (apiCall.auth === "SessionTicket")
+        return tabbing + "if PlayFabSettings.ClientSessionTicket == None:\n"+tabbing+"    raise Exception(PlayFabExceptionCode.NotLoggedIn, \"Must be logged in to call this method\")\n";
+    if (apiCall.auth === "SecretKey")
+        return tabbing + "if PlayFabSettings.DeveloperSecretKey == None:\n"+tabbing+"    raise Exception(PlayFabExceptionCode.DeveloperKeyNotSet, \"Must have DeveloperSecretKey set to call this method\")\n";
+    if (apiCall.url === "/Authentication/GetEntityToken")
+        return tabbing + "authKey = None\n"+tabbing+"authValue = None\n"
+            + tabbing + "if PlayFabSettings.ClientSessionTicket != None:\n "+tabbing+"    authKey = \"X-Authorization\"\n"+tabbing+"    authValue = PlayFabSettings.ClientSessionTicket \n"
+            + tabbing + "if PlayFabSettings.DeveloperSecretKey != None:\n "+tabbing+"    authKey = \"X-SecretKey\"\n"+tabbing+"    authValue = PlayFabSettings.DeveloperSecretKey \n"
+            + tabbing + "if PlayFabSettings.EntityToken != None:\n "+tabbing+"    authKey = \"X-EntityToken\"\n"+tabbing+"    authValue = PlayFabSettings.EntityToken\n";
+    return "";
+}
+
+function getResultActions(tabbing, apiCall, api) {
+    if (apiCall.result === "LoginResult")
```

Comment:
This isn't saving the tokens to PlayFabSettings as expected.
I'll come help you with it.